sdk/src/transaction/sanitized.rs

```diff
@@ -294,3 +297,57 @@ impl SanitizedTransaction {
         }
     }
 }
+
+#[cfg(test)]
+#[allow(clippy::integer_arithmetic)]
+mod tests {
+    use {
+        super::*,
+        crate::signer::{keypair::Keypair, Signer},
+        solana_program::vote::{self, state::Vote},
+    };
+
+    #[test]
+    fn test_try_create_simple_vote_tx() {
```

Comment:
`is_simple_vote_tx` is an optional argument to the `try_create` fn.

Logically, these shouldn't ever be different. But the behavior is currently the passed value takes precedence.

Should the test(s) make sure that is the case?